web/package.json

```diff
@@ -88,6 +88,17 @@
     "ignorePatterns": [
       "!.storybook"
     ],
+    "rules": {
+      "import/no-extraneous-dependencies": [
```

Comment:
Should we probably disable that rule for all files?
Because in a `theo-import.js` file I also disabled this rule - https://github.com/ubergrape/grape-ds/blob/master/web/src/tokens/theo-import.js#L2

Reply:
done!